src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -4,6 +4,7 @@ Release History
 ===============
 0.1.26
 ++++++
+* fix backup/restore
 * minor changes
```

Comment:
You should remove minor changes since you actually have a custom-facing fix for the release (as opposed to internal or help fixes which is what minor fixes means).  Also, recommend more explanation in the notes, as this is what communicates to customers.
"* `webapp backup/restore`: Fix issue where ...`